django_project/frontend/src/pages/Admin/ContextLayer/Form/index.jsx

```diff
@@ -192,8 +225,8 @@ export default function ContextLayerForm() {
               useOverrideLabel={false}
             />
           ),
-          'Fields': <div/>,
-          'Label': <div/>,
+          'Fields': <div />,
```

Comment:
Could you change the rule to prevent this changes?

Reply:
done